### PR TITLE
chore: add cSpell commit hook

### PR DESCRIPTION
**Pull request by nschonni**
Since it is now being run on all files, it can be run as part of the pre-commit hook to check locally

### CODE REVIEW

**Comment by alexander-akait** [comment]
@nschonni sorry for delay, can you rebase?

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "prepare": "husky install"
   },
   "lint-staged": {
+    "*": "cspell",
```
Comment (alexander-akait) [inline package.json:65]: I think we need `--no-must-find-files` here

Comment (nschonni) [reply package.json:65]:
```suggestion
    "*": "cspell --no-must-find-files",
```

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thank you

**Pull request merged**